test/integration/ordinary/itest_slc.py

```diff
@@ -87,12 +101,12 @@ def test_export_slc(sample_slc: ScriptLanguageContainer):
     sample_slc.export()
     export_path = sample_slc.workspace.export_path
     assert export_path.exists()
-    tgz = [f for f in export_path.glob("*.tar.gz")]
-    assert len(tgz) == 1
-    assert tgz[0].is_file()
-    tgz_sum = [f for f in export_path.glob("*.tar.gz.sha512sum")]
-    assert len(tgz_sum) == 1
-    assert tgz_sum[0].is_file()
+    tar = [f for f in export_path.glob("*.tar")]
+    assert len(tar) == 1
+    assert tar[0].is_file()
+    tar_sum = [f for f in export_path.glob("*.tar.sha512sum")]
+    assert len(tar_sum) == 1
+    assert tar_sum[0].is_file()
```

Comment:
If changing the compression to use in class `ScriptLanguageContainer` then this tests probably needs to be updated, too.

Reply:
Fixed